datastore/grantserver/wallet.go

```diff
@@ -0,0 +1,38 @@
+package grantserver
+
+import (
+	"github.com/brave-intl/bat-go/wallet"
+	uuid "github.com/satori/go.uuid"
+)
+
+// InsertWallet inserts the given wallet
+func (pg *Postgres) InsertWallet(wallet *wallet.Info) error {
+	// NOTE on conflict do nothing because none of the wallet information is updateable
+	statement := `
+	insert into wallets (id, provider, provider_id, public_key)
+	values ($1, $2, $3, $4)
+	on conflict do nothing
+	returning *`
+	_, err := pg.DB.Exec(statement, wallet.ID, wallet.Provider, wallet.ProviderID, wallet.PublicKey)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// GetWallet by ID
+func (pg *Postgres) GetWallet(ID uuid.UUID) (*wallet.Info, error) {
+	statement := "select * from wallets where id = $1"
+	wallets := []wallet.Info{}
+	err := pg.DB.Select(&wallets, statement, ID)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(wallets) > 0 {
```

Comment:
Looking at the sqlx documentation If we use `pg.DB.Get` it returns a singular item. Might be better to leverage it?

```
    // You can also get a single result, a la QueryRow
    jason = Person{}
    err = db.Get(&jason, "SELECT * FROM person WHERE first_name=$1", "Jason")
```

Reply:
good find! can you make a follow up issue to refactor - this was just moved and I'd rather we address all such instances in one go